### DISCUSSION TITLE
TypeError: fetch failed (external api) - only locally on Windows

### DISCUSSION THREAD

**Discussion by ESEBUR1** [vercel/next.js#55738] [Help]
### Link to the code that reproduces this issue

https://github.com/ESEBUR1/test

### To Reproduce

1. Start the application in development on Windows 10 Enterprise
2. Homepage crash already

### Current vs. Expected behavior

It should fetch correctly the data as it does when I deploy to Vercel or when I try on my Mac device.
Here it is working correctly:

https://test-etonxq927-esebur1.vercel.app/

But locally that's all I see

![Screenshot 2023-09-21 120316](https://github.com/vercel/next.js/assets/144676072/6c67f347-f009-4e60-8ea4-cfb86fc8cc1c)

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: win32
      Arch: x64      
      Version: Windows 10 Enterprise
    Binaries:
      Node: 16.20.2
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant Packages:
      next: 13.4.20-canary.40
      eslint-config-next: 13.4.19
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.2.2
    Next.js Config:
      output: N/A
```

### Which area(s) are affected? (Select all that apply)

Data fetching (gS(S)P, getInitialProps)

**Comment by balazsorban44** [comment]
I could not reproduce this on my end, but try upgrading Node.js.

You are on v16, which is now [end-of-life](https://endoflife.date/nodejs). It's unclear if you were on the same versions on the other machine(s).

There might be more info in your terminal logs, but from the screenshot, it's not clear.

**Reply by ESEBUR1** [reply]
I could finally update node version to v18.17.1 and still get the same error, even if I do fetch from a local json file @balazsorban44
